kolibri/plugins/coach/assets/src/views/exams-page/change-exam-visibility-modal.vue

```diff
@@ -129,26 +134,26 @@
             assignment => assignment.assignmentId
           );
           this.updateExamAssignments(this.examId, classCollection, groupAssignments);
-        } else if (this.groupsSelected.length) {
+        } else if (this.selectedGroups.length) {
           const unassignGroups = this.initiallySelectedGroups().filter(
-            initialGroup => !this.groupsSelected.find(newGroup => newGroup.id === initialGroup.id)
+            initialGroup => !this.selectedGroups.find(newGroup => newGroup === initialGroup)
```

Comment:
Since the result of `this.selectedGroups.find` is being coerced to Boolean, it might be more appropriate to use `this.selectedGroups.includes`, which returns a Boolean off the bat.

Reply:
(and we now polyfill `includes`)